dbld/Makefile.am

```diff
@@ -9,23 +9,27 @@ EXTRA_DIST += \
 	dbld/rules \
 	dbld/bootstrap	\
 	dbld/images/artful.dockerfile \
-	dbld/images/centos6.dockerfile \
-	dbld/images/centos7.dockerfile \
 	dbld/images/devshell-apt \
 	dbld/images/devshell.dockerfile \
 	dbld/images/devshell-pip \
 	dbld/images/helpers/entrypoint.sh \
 	dbld/images/helpers/fake-sudo.sh \
 	dbld/images/helpers/functions.sh \
 	dbld/images/helpers/gosu.pubkey \
-	dbld/images/jessie.dockerfile \
 	dbld/images/README.md \
 	dbld/images/required-apt \
 	dbld/images/required-epel \
 	dbld/images/required-obs \
 	dbld/images/required-pip \
 	dbld/images/required-yum \
-	dbld/images/trusty.dockerfile \
-	dbld/images/xenial.dockerfile \
+	dbld/images/centos_6.dockerfile \
+	dbld/images/centos_7.dockerfile \
+	dbld/images/debian_8_jessie.dockerfile \
+	dbld/images/debian_9_stretch.dockerfile \
+	dbld/images/debian_10_buster.dockerfile \
+	dbld/images/ubuntu_1404_trusty.dockerfile \
+	dbld/images/ubuntu_1604_xenial.dockerfile \
+	dbld/images/ubuntu_1804_bionic.dockerfile \
+	dbld/images/ubuntu_1810_cosmic.dockerfile \
 	dbld/make \
```

Comment:
I would either use the code name (e.g. xenial) or the version number in the names of files but not both. If you ask me, I'd prefer names, but YMMV.